test/modules/Modules.Test/TestResultCoordinator/Reports/DirectMethod/DirectMethodConnectivityReportDataWithSenderAndReceiverSource.cs

```diff
@@ -185,7 +203,7 @@ class DirectMethodConnectivityReportDataWithSenderAndReceiverSource
                     // Non-Offline test
                     Enumerable.Range(1, 7).Select(v => (ulong)v),
                     Enumerable.Range(1, 7).Select(v => (ulong)v),
-                    new List<HttpStatusCode> { HttpStatusCode.OK, HttpStatusCode.OK, HttpStatusCode.NotFound, HttpStatusCode.OK, HttpStatusCode.OK, HttpStatusCode.OK, HttpStatusCode.OK },
+                    new List<HttpStatusCode> { HttpStatusCode.NotFound, HttpStatusCode.NotFound, HttpStatusCode.NotFound, HttpStatusCode.NotFound, HttpStatusCode.OK, HttpStatusCode.OK, HttpStatusCode.OK },
```

Comment:
I wanted to keep the test functioning as expected, ignoring any tolerances. Therefore I had to exceed the tolerance (by adding more errors) so that it would surpass the new threshold.